### PR TITLE
Fix #2567 Include all options in excluded item list

### LINKED ISSUE #2567: Include all albums option in excluded items list
**Actual Behaviour**

In Excluded items activity, if we want to remove all albums we have to remove them one by one.

**Expected Behaviour**

There should be an option to include all the excluded items at once, so that user don't have to include them one by one.

**Screenshots of the issue**
image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by theabhishekavi**
Fixed #2567 

Changes: Restore all excluded items directly instead of selecting one by one

Screenshots of the change: 
image

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java [hunk 1]
```diff
@@ -49,6 +52,30 @@ private void checkNothing() {
         a.setVisibility(excludedFolders.size() == 0 ? View.VISIBLE : View.GONE);
     }
 
+    public void restoreAll(int itemcount){
+        if (excludedFolders.size()!=0){
+            for (int i=0;i<itemcount;i++){
+                h.clearAlbumExclude(excludedFolders.remove(0).getAbsolutePath());
+            }
+            Toast.makeText(ExcludedAlbumsActivity.this,"All folders are restored!",Toast.LENGTH_SHORT).show();
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java]: Replace this with a snackbar.

Comment (theabhishekavi) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java]: Okay.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java [hunk 2]
```diff
@@ -49,6 +56,35 @@ private void checkNothing() {
         a.setVisibility(excludedFolders.size() == 0 ? View.VISIBLE : View.GONE);
     }
 
+    public void restoreAll(int itemcount){
+        if (excludedFolders.size()!=0){
+            for (int i=0;i<itemcount;i++){
+                h.clearAlbumExclude(excludedFolders.remove(0).getAbsolutePath());
+            }
+            SnackBarHandler.show(findViewById(R.id.rl_ea), "All folders are restored", Snackbar.LENGTH_SHORT).show();
+            new Handler().postDelayed(new Runnable() {
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java]: What is the use of the postDelayed handler and finish() ?

Comment (theabhishekavi) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java]: as soon as all the folders are restored, its better not to show the blank screen

Comment (yashk2000) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java]: ok

**Comment by yashk2000** [comment]
Remove the unused imports.

**Comment by theabhishekavi** [comment]
@mariobehling @abishekvashok @sauravvishal8797 kindly review :)

**Pull request merged**